src/quo/components/avatars/user_avatar/schema.cljs

```diff
@@ -0,0 +1,21 @@
+(ns quo.components.avatars.user-avatar.schema)
+
+(def ^:private ?profile-picture-fn-params
+  [:map
+   [:length :int]
+   [:full-name :string]
+   [:font-size :int]
+   [:indicator-size {:optional true} [:maybe :int]]
+   [:indicator-color {:optional true} [:maybe :string]]
+   [:override-theme :schema.common/theme]
```

Comment:
we certainly don't want override theme props anymore, can you change this to `theme`? 
probably doc string was outdated.

Reply:
I think in this case **profile-picture-fn** will expect `override-theme`. For example, the **profile-picture-fn** could be the value returned from the [`get-initials-avatar-uri-fn`](https://github.com/status-im/status-mobile/blob/28f43acb833e302dbd0d013ebe4c8e1cce538e6a/src/utils/image_server.cljs#L199): https://github.com/status-im/status-mobile/blob/28f43acb833e302dbd0d013ebe4c8e1cce538e6a/src/utils/image_server.cljs#L199-L230

Reply:
Ab okay, that's also some code that needs adjustment. We moved away from override theme approach a while back

Reply:
I think in this case there are some re-frame subs that initialise the `profile-picture-fn` with a `:theme`, and then later some rendering logic is allowed override that initial theme. 

This makes me wonder:
* Should I rename the field `override-theme` to `theme`?
* Or should I refactor the code to remove the overriding logic?
  * Is there a clear preference on who should control the theme?
  
Depending on the changes that should be made, I may split them into a separate PR, lmk your thoughts when you can 🙏

Reply:
sounds like we should do it in a separate pr/issue, could you create an issue to track this? 🙏

Reply:
yup yup will do 👍